collectors/ps_constants/tasks.py

```diff
@@ -0,0 +1,72 @@
+"""
+product definitions collector
+"""
+from celery.schedules import crontab
+from celery.utils.log import get_task_logger
+from django.utils import timezone
+
+from collectors.framework.models import collector
+from osidb.models import SpecialConsiderationPackage, UbiPackage
+
+from .constants import PS_CONSTANTS_REPO_BRANCH, PS_CONSTANTS_REPO_URL
+from .core import (
+    fetch_ps_constants,
+    sync_special_consideration_packages,
+    sync_ubi_packages,
+)
+
+logger = get_task_logger(__name__)
+
+
+# GitLab URL to specific branch
+PS_CONSTANTS_BASE_URL = "/".join(
+    (
+        PS_CONSTANTS_REPO_URL,
+        "-",
+        "raw",
+        PS_CONSTANTS_REPO_BRANCH,
+        "data",
+    )
+)
+
+
+@collector(
+    # Execute this every 6 hours
+    # TODO: crontab seems to be not sufficient as a scheduler here
+    # since it is only capable of running the job at every fixed third hour
+    # eg. 3:00,8:00,etc. and thus there exist a scenario in which
+    # the OSIDB is run lets say 3:01 and this job will be scheduled on 8:49
+    # which is really not what we want, since there may be other collectors
+    # depending on this one, odd minute number was chosen in order to not have multiple
+    # tasks running at the same time
+    # TODO: Use django_celery_beat which has PeriodicTask with IntervalSchedule
+    #  What we use here is equivalent to PeriodicTask with CrontabSchedule
```

Comment:
This is a known issue (it probably even has a Jira) and can make problems during the very initial sync. The set collect period or the timing should be ideally ignored at such situation and everything should be synced immediately going from the models no dependencies and progressing with others when their dependencies get resolved. If we only have time for it :smile: (not complaining about your code here, so you can ignore this note)